### DISCUSSION TITLE
how to reduce request count

### DISCUSSION THREAD

**Discussion by ranjit-git** [projectdiscovery/nuclei#5656] [General]
```
echo https://example.com|nuclei -id "akamai-detect,angular-detect" -dreq
```
this command  will send 2 request to https://example.com with same path and then match the keyword .
Is there any way that this request will be  sent only once because both request is same and then match the response keyword ?
How to optimize this?

**Comment by ehsandeep** [comment]
@ranjit-git this is done automaticlaly by nuclei, in this case both templates does not send same request,  [akamai-detect](https://cloud.projectdiscovery.io/?template=akamai-detect) sends additional header with`Pragma: akamai-x-cache-on`

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
